src/ewatercycle/observation/usgs.py

```diff
@@ -106,7 +67,7 @@ def get_usgs_data(station_id, start_date, end_date, parameter="00060", cache_dir
             [float(point.members[0]["value"]) / 35.315 for point in station.elements],
             dtype=np.float32,
         )
-        times = [point.time for point in station.elements]
+        times = pd.to_datetime([point.time for point in station.elements])
 
         attrs = {
             "units": "cubic meters per second",
```

Comment:
Why not also do `m3/s` here?
```suggestion
            "units": "m3/s",
```